internal/csi/csi.go

```diff
@@ -50,18 +52,33 @@ func NewTester(cfg *ec2config.Config, terminateOnExit, journalctlLogs bool) (tes
 	return tester, nil
 }
 
-func CreateConfig(vpcID, branchOrPR string) *ec2config.Config {
-	cfg := ec2config.NewDefault()
+func CreateConfig(vpcID, prNum, githubAccount, githubBranch string) (cfg *ec2config.Config, err error) {
+	cfg = ec2config.NewDefault()
 	cfg.UploadTesterLogs = false
 	cfg.VPCID = vpcID
 	cfg.IngressRulesTCP = map[string]string{"22": "0.0.0.0/0"}
 	cfg.Plugins = []string{
 		"update-amazon-linux-2",
 		"install-go-1.11.3",
-		"install-csi-" + branchOrPR,
+	}
+	// If prNum is set to an non-empty string, the master branch of kubernetes-sigs/aws-ebs-csi-driver is used,
+	// regardless of whether or not githubAccount and githubBranch have different values.
+	if prNum != "" {
+		if githubAccount != "kubernetes-sigs" || githubBranch != "master" {
+			fmt.Printf("WARNING: PR number %s takes precedence over non-default GitHub account and/or branch.\n", prNum)
+		}
+		cfg.Plugins = append(cfg.Plugins, "install-csi-"+prNum)
+	} else {
+		cfg.CustomScript, err = createCustomScript(gitAccountAndBranch{
+			Account: githubAccount,
+			Branch:  githubBranch,
+		})
+		if err != nil {
+			return nil, err
+		}
 	}
 	cfg.Wait = true
-	return cfg
+	return cfg, nil
 }
 
 func createPermissions(cfg *ec2config.Config) (resources *iamResources, err error) {
```

Comment:
Is this going to be a null pointer bug related to https://github.com/aws/aws-k8s-tester/pull/30?